webapp/src/components/App/Studies/StudyCard.tsx

```diff
@@ -450,20 +470,23 @@ const StudyCard = memo((props: Props) => {
           onClose={handleMenuClose}
         >
           {study.archived ? (
-            <MenuItem onClick={handleUnarchiveClick}>
-              <ListItemIcon>
-                <UnarchiveOutlinedIcon
-                  sx={{
-                    color: "action.active",
-                    width: "24px",
-                    height: "24px",
-                  }}
-                />
-              </ListItemIcon>
-              <ListItemText>{t("global.unarchive")}</ListItemText>
-            </MenuItem>
+            <>
```

Comment:
Avoid using a Fragment on Menu component, you can replace it by a div or Box. 
see: 
- https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Menu/Menu.js#L134
- https://github.com/mui/material-ui/issues/16181

their is another existing Fragment in the Menu component below your added code, could you please also remove it ?